docs/docs/webui.md

```diff
@@ -19,7 +20,7 @@ There are also two included example YAML files that make use of this configurati
 * [config/local_scheduler.yml](/mesos/chronos/blob/master/config/local_scheduler.yml#L4)
 * [config/local_scheduler_nozk.yml](/mesos/chronos/blob/master/config/local_scheduler_nozk.yml#L7)
 
-## Build Requirements
+### Build Requirements
 
```

Comment:
So I experienced errors when I tried to compile the Chronos UI, specifically I got errors when in the minification step, some time back on my own machine (16 GB Memory). I think we can keep this in the documentation for now. And then we will fix these problems, when will start to rebuild the UI.